source/frontend/src/assets/scss/index.scss

```diff
@@ -18,6 +18,68 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
 }
 
+$spacer: 1.6rem;
+$spacers: ();
+// stylelint-disable-next-line scss/dollar-variable-default
+@for $i from 1 through 50 {
```

Comment:
not that this is a bit of a change from current state, which seems to provide the following:
p-1: .25rem -> .4rem
p-2: .5rem -> .8rem
p-3: 1rem -> 1.2 rem
p-4: 1.5rem -> 1.6 rem

theoretically this is ok, because it will force conformance of all existing components, but will cause many spacing changes across the application that may cause line breaks, etc.